### PR TITLE
Add tooltip

### LINKED ISSUE #1: Display a card on hover
Like GwentDB, but on any authorised page.

Questions:

- Use gwentapi or use gwentdb ?
- Cross-origin and XSS
- Avoid CSS conflicts
- How to style the text in the document? Preserve existing links?

### PR DESCRIPTION
**Pull request by Zhouzi**
The idea is to offer a function that accepts two arguments: a target DOM element and the related card.

Close #1

### CODE REVIEW

**Comment by Zhouzi** [comment]
Almost done, just need to clean a thing or two and set up a decent page.

**Comment by Zhouzi** [comment]
I'll rather design the GitHub page in a separate PR. This one improves the build system and structure so it's better to merge it sooner than later.

Right now, here's what it looks like:



Don't pay too much attention to the page itself, the tooltip is what matters. I'd love to use a custom font, Arial's not so pretty but that would probably be an overkill. Or maybe it wouldn't be too bad?

**Comment by Soreine** [comment]
We can always use some cool system fonts if available (Helvetica)

File: src/core/tooltip/tooltip.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import { createElement } from 'jsx-dom';
+import styles from './tooltip.css';
+
+function createTooltip(card, target) {
+  const wrapper = <hyper-gwent-tooltip style={{
+    display: 'none',
+    position: 'fixed',
+    transform: 'translate(10px, -60px)',
+    zIndex: 999999999,
```
Comment (Zhouzi) [inline src/core/tooltip/tooltip.js]: Yep, I went over the limit on this one 😄  The browser is normalizing to maximum value but a good old `10` could have been enough.

**Comment by Zhouzi** [comment]
I'll probably tweak the design again in the future, I had some other ideas.

**Pull request merged**